cli/script/command-executor.ts

```diff
@@ -1180,48 +939,12 @@ function isBinaryOrZip(path: string): boolean {
         || path.search(/\.ipa$/i) !== -1;
 }
 
-function throwForMissingCredentials(accessKeyName: string, providerName: string, providerUniqueId: string): void {
-    if (!accessKeyName) throw new Error("Access key is missing.");
-    if (!providerName) throw new Error("Provider name is missing.");
-    if (!providerUniqueId) throw new Error("Provider unique ID is missing.");
-}
-
-function throwForInvalidAccessKeyId(accessKeyId: string, accessKeyName: string): void {
-    if (!accessKeyId) {
-        throw new Error("Access key \"" + accessKeyName + "\" does not exist.");
-    }
-}
-
-function throwForInvalidApp(app: App, appName: string): void {
```

Comment:
Verify that error messages are unchanged - as we are resolving names on the server, an invalid name will result in an error coming from the server rather than the CLI.